subgraph/subgraph.yaml

```diff
@@ -47,36 +24,15 @@ templates:
         - name: IVaultSponsoring
           file: ../artifacts/contracts/vault/IVaultSponsoring.sol/IVaultSponsoring.json
       eventHandlers:
-        - event: DepositMinted(indexed uint256,uint256,uint256,uint256,indexed address,indexed address,uint256,uint256,bytes)
+        - event: DepositMinted(indexed uint256,uint256,uint256,uint256,indexed address,indexed address,uint256,uint64,bytes)
           handler: handleDepositMinted
         - event: DepositBurned(indexed uint256,uint256,indexed address)
           handler: handleDepositBurned
         - event: Sponsored(indexed uint256,uint256,indexed address,uint256)
           handler: handleSponsored
         - event: Unsponsored(indexed uint256)
           handler: handleUnsponsored
-        - event: YieldClaimed(uint256,indexed address,uint256,uint256)
+        - event: YieldClaimed(uint256,indexed address,uint256,uint256,uint256)
           handler: handleYieldClaimed
         - event: TreasuryUpdated(indexed address)
           handler: handleTreasuryUpdated
-
-  - name: Claimers
```

Comment:
Why are we removing this?

Reply:
could be related to removal of IClaimers here https://github.com/lindy-labs/sc_solidity-contracts/pull/57

Reply:
I see, but we still need this mapping if we are going to keep track of who owns which claimer wallet unless we remove that functionality.

Reply:
My bad thought could be removed, will try to add without the IClaimers abi